pages/directory/[orgName].tsx

```diff
@@ -34,14 +33,6 @@ const OrganizationPage = () => {
           </div>
         </div>
         <div className="flex md:grid md:grid-cols-6 gap-4 p-10">
-          <div className="md:col-span-2">
-            <Image
-              src={org.imageLink}
-              alt={org.name}
-              width={250}
-              height={250}
-            />
-          </div>
           <div className="md:col-span-2">
             <p>
               Lorem ipsum dolor sit amet consectetur adipisicing elit. Esse
```

Comment:
Could this potentially use the org.description field from Firestore also?

Reply:
Do you mean like writing that inside of the p tag?